packages/shared/lib/services/sync/orchestrator.service.ts

```diff
@@ -174,8 +176,13 @@ export class Orchestrator {
         providerConfigKey: string,
         syncNames: string[],
         command: SyncCommand,
-        connectionId?: string
+        connectionId?: string,
+        fullResync?: boolean
     ): Promise<ServiceResponse<boolean>> {
+        if (fullResync && command !== SyncCommand.RUN) {
```

Comment:
would it make sense to have 2 commands `RUN` and `RUN_FULL`?

Reply:
Agree, the `RUN_FULL` command should be used here: https://github.com/NangoHQ/nango/blob/f718b2ce8edd70de998affc4a85a10475e2b0cee/packages/shared/lib/models/Sync.ts#L258

Reply:
There is already a RUN_FULL command that seems not implemented (or at least I didn't get it) so I was not comfortable using it. Maybe @khaliqgant knows better